### ISSUE TITLE
Quantize: lacking min/max data, when i transform the quantize graph to tflite

### ISSUE THREAD

**Issue by 1icas** [tensorflow/tensorflow#20605]
tensorflow version: 1.8
model: my model structure is very simliar to mobilenet-v1
desc: I trained my model use the tf.contrib.quantize.create_training_graph function, and call the tf.contrib.quantize.create_eval_graph() when i generate the eval graph.
And then I use 
bazel-bin/tensorflow/contrib/lite/toco/toco \
--input_file=/home/admin_pc/model_test/output_quant.pb \
--output_file=/home/admin_pc/model_test/mobilenet_qg.tflite \
--input_fromat=TENSORFLOW_GRAPHDEF \
--output_format=TFLITE \
--inference_type=QUANTIZED_UINT8 \
--input_array=image \
--output_array=Openpose/MConv_Stage3_L2_5_pointwise/BatchNorm/FusedBatchNorm \
--input_shape=1,224,224,3 \
--change_concat_input_ranges=false \
--std_value=128 --mean_value=128
to generate the tflite

but some errores occurs:

2018-07-07 12:34:56.116401: I tensorflow/contrib/lite/toco/graph_transformations/graph_transformations.cc:39] After pre-quantization graph transformations pass 1: 58 operators, 169 arrays (1 quantized)
2018-07-07 12:34:56.117540: F tensorflow/contrib/lite/toco/tooling_util.cc:1581] Array MobilenetV1/Conv2d_1_depthwise/depthwise, which is an input to the Conv operator producing the output array MobilenetV1/Conv2d_1_pointwise/Relu6, is lacking min/max data, which is necessary for quantization. Either target a non-quantized output format, or change the input graph to contain min/max information, or pass --default_ranges_min= and --default_ranges_max= if you do not care about the accuracy of results.
Aborted (core dumped)

**Comment by suharshs** [comment]
Could you be more specific on what operations you are talking about? Is this for tensorflow lite's quantization or the tensorflow operations?

If this is for tensorflow, the relu6 operations have the quantization range fixed at 0 and 6 which helps quantize to good accuracy. Relu can also be quantized but needs the quantization range to be known. This can be accomplished with TensorFlow Lite's quantized models https://www.tensorflow.org/performance/quantization 

TFLite supports quantization of ReLU and ReLU6 so you should consider using that.

**Comment by 1icas** [comment]
I am very sorry to bother you. i will modify my question soon.

**Comment by 1icas** [comment]
@suharshs looking forward to your reply

**Comment by 1icas** [comment]
it's all of my faults

**Comment by suharshs** [comment]
Out of curiosity, what was the issue?

**Comment by 1icas** [comment]
well,I don't know it's a bug or my fault when i  roughed read the paper (Quantization and Training of Neural Networks for Efficient Integer-Arithmetic-Only Inference) today . I will describe this problem detailed.
standard mobilenet-v1 structure:  depthwise conv - bn - relu - pointwise conv - bn - relu
my model structure: depthwise conv - pointwise conv - bn - relu 
So, i changed my model structure to the standard mobilenet-v1 structure.  Problem solved.
It looks like that i need the standard mobilenet-v1 structure which i want to quantize the model.
I read the paper 'Quantization and Training of Neural Networks for Efficient Integer-Arithmetic-Only Inference' today. The paper said that we can quantize the  conv layer without a bn. 
So it's a bug or that we must add a bn layer after the conv layer which we want to quantize.
@suharshs looking forward to your reply.

**Comment by 1icas** [comment]
@suharshs can you answer my question ? T.T
